proto/raw-js-openapi/package.json

```diff
@@ -9,10 +9,11 @@
   "devDependencies": {
     "ts-loader": "^6.0.1",
     "typescript": "3.9.7",
-    "webpack": "^5.38.1",
+    "webpack": "^5.76.0",
```

Comment:
Forgive my dumb questions here, but why 5.76.0 and not one of the patches?

Reply:
Only because my experience with these tools is limited, and the output does what we need with this version.

Reply:
Ok - should I update package.json then, or is it sufficient to say "the combination of the committed package.json and package-lock.json result in a deterministic build"?

Reply:
No need to update, the combination of package.json/package-lock.json results in a deterministic build.